alerta/auth/basic_ldap.py

```diff
@@ -0,0 +1,72 @@
+
+import logging
+from uuid import uuid4
+
+from flask import current_app, request, jsonify, render_template
+from flask_cors import cross_origin
+
+from alerta.auth.utils import is_authorized, create_token, get_customers
+from alerta.exceptions import ApiError
+from alerta.models.user import User
+from alerta.utils.api import absolute_url
+from . import auth
+
+import ldap
+
+@auth.route('/auth/signup', methods=['OPTIONS', 'POST'])
+@cross_origin(supports_credentials=True)
+def signup():
+    raise NotImplementedError
```

Comment:
This might not be ideal, however there is no good way to handle signup here yet without a change in https://github.com/alerta/angular-alerta-webui/